src/components/AddressButton/AddressButton.tsx

```diff
@@ -102,25 +125,28 @@ interface StyledLinkProps {
   overflow?: string;
 }
 
+interface StyledAddressProps {
+  darkMode?: boolean;
+  color?: string;
+  overflow?: string;
+}
+
 interface StyledCopyProps {
   darkMode?: boolean;
   uniswap?: boolean;
 }
 
-const StyledButton = styled.div<StyledButtonProps>`
+const StyledButton = styled.a<StyledButtonProps>`
   background: ${(props) =>
     props.darkMode
-      ? "radial-gradient(circle at top,hsl(339deg 17% 15% / 100%),hsl(339deg 20% 10% / 100%))"
-      : "radial-gradient(circle at top,hsl(338deg 20% 96% / 100%),hsl(338deg 20% 94% / 100%))"};
+      ? "radial-gradient(circle at top,hsl(339deg 17% 15% / 20%),hsl(339deg 20% 10% / 20%))"
+      : "radial-gradient(circle at top,hsl(360deg 0% 100% / 100%),hsl(360deg 0% 95% / 100%))"};
   box-shadow: ${(props) =>
     props.darkMode
-      ? "-8px 8px 16px 0 hsl(339deg 20% 5% / 100%), 8px -8px 16px 0px hsl(339deg 100% 100% / 7.5%)"
-      : "-8px 8px 16px 0 hsl(338deg 95% 4% / 15%), 8px -8px 16px 0px hsl(338deg 100% 100% / 100%);"};
-  -webkit-align-items: center;
-  -webkit-box-align: center;
-  -ms-flex-align: center;
+      ? "-1px 1px 1px 1px hsl(339deg 20% 5% / 100%), 1px -1px 1px 1px hsl(339deg 20% 5% / 7.5%)"
+      : "-1px 1px 1px 1px hsl(339deg 20% 5% / 25%), 1px -1px 1px 1px hsl(339deg 20% 5% / 7.5%);"};
```

Comment:
the width looks good, but [these shadows](https://deploy-preview-273--epic-goldberg-1d6f10.netlify.app/#/addresses) doesnt, please revert